HHAuto.user.js

```diff
@@ -3859,8 +3860,7 @@ function goAndCollectDailyGoals()
                 }
             }
 
-
-            if (buttonsToCollect.length >0)
+            if (buttonsToCollect.length >0 || potionsNum <100)
```

Comment:
Strange to enter in this part when there is no button but I understand.

Reply:
I wanted to still have 30mins timer for daily goals re-check:  there could be no 'claim' buttons but the last goal is still not achieved. the exit on the 30min cycle is when last goal will be collected (no Claim button available and potions over 100)